recipes/mapbox_earcut/0001-removed-setup_requires.patch

```diff
@@ -0,0 +1,24 @@
+From 7a7e28354f659b11a94dd64eb9a2b23f4d4c520e Mon Sep 17 00:00:00 2001
+From: Chris Barker <Chris.Barker@noaa.gov>
+Date: Fri, 2 Oct 2020 16:15:25 -0700
+Subject: [PATCH] removed setup_requires
+
+---
+ setup.py | 1 -
+ 1 file changed, 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index 84cc14e..14ef749 100644
+--- a/setup.py
++++ b/setup.py
+@@ -88,7 +88,6 @@ setup(
+     long_description_content_type='text/markdown',
+     license='ISC',
+     ext_modules=[CMakeExtension('mapbox_earcut')],
+-    setup_requires=['setuptools_scm'],
```

Comment:
Is this really needed?

Reply:
it was -- I think setup_rquires was breaking the build to to the lovely easy_install. There may be another work around, but it's not actually used anymore -- so not an actual dependency.

I submitted an issue:
https://github.com/skogler/mapbox_earcut_python/issues/4

which has not been addressed yet.

Reply:
@ChrisBarker-NOAA, can you remove this patch? I think the build error you got was because of some other issue